open-metadata-implementation/admin-services/docs/user/configuring-the-omag-server-platform.md

```diff
@@ -38,8 +38,8 @@ its **application.properties** file found in the **resources** subdirectory.
 * Defining the port that the OMAG Server Platform will listen on for
   REST API calls.
   
-* Controlling the level of developer logging that the platform
-  produces when it is running.
+* Controlling the level of technical logging that the platform
```

Comment:
What does "technical" mean - I prefer to use "developer" since that is the audience that this logging is suitable for.

Reply:
I came up with "techical" exactly because I was trying to distinguish from the AuditLog but now I see and agree with you that it is not meaninful from user perspective.